docs/source/registrationOverview.rst

```diff
@@ -209,6 +211,17 @@ within each voxel, the latter uses the centers of all N voxels. Thus, for
 repeated random sampling with 100% rate, you get different samples and likely
```

Comment:
Avoid second person. "you get different sample"-> "different samples are produced"

Reply:
done.